packages/fork-choice/src/forkChoice/interface.ts

```diff
@@ -43,18 +44,18 @@ export type AncestorResult =
 
 // Reason for not proposer boost reorging
 export enum NotReorgedReason {
-  HeadBlockIsTimely,
-  ParentBlockNotAvailable,
-  ProposerBoostReorgDisabled,
-  NotShufflingStable,
-  NotFFGCompetitive,
-  ChainLongUnfinality,
-  ParentBlockDistanceMoreThanOneSlot,
-  ReorgMoreThanOneSlot,
-  ProposerBoostNotWornOff,
-  HeadBlockNotWeak,
-  ParentBlockIsStrong,
-  NotProposingOnTime,
```

Comment:
I think using string would be better in case we want to debug later through metrics